presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -224,11 +220,7 @@ public static void createScriptFile(String filePath, String scriptContent)
             parentDir.mkdirs();
         }
 
-        if (file.exists()) {
-            throw new IOException("File exists: " + filePath);
-        }
-
-        try (OutputStream output = new FileOutputStream(file);
+        try (OutputStream output = new FileOutputStream(file, false);
```

Comment:
```suggestion
        try (OutputStream output = new FileOutputStream(file);
```

Aren't these equivalent?